chef/cookbooks/glance/templates/default/glance-cache.conf.erb

```diff
@@ -47,7 +47,7 @@ registry_port = <%= @registry_bind_port %>
 #                glance.store.sheepdog.Store,
 #                glance.store.cinder.Store,
 #                glance.store.vmware_datastore.Store,
-known_stores = glance.store.filesystem.Store,glance.store.http.Store,glance.store.rbd.Store,glance.store.swift.Store,glance.store.cinder.Store,glance.store.vmware_datastore.Store
+known_stores = <%= @glance_stores %>
```

Comment:
@tboerger, @vuntz : There is no problem with using array here. At least we have to remove `glance.store.vmware_datastore.Store` when it is not used, because it cause that glance-scrubber is crashing.